tests/gpflow/conditionals/test_multioutput.py

```diff
@@ -317,29 +317,104 @@ def test_sample_conditional_mixedkernel():
     )
 
 
-@pytest.mark.parametrize(
-    "func, R",
-    [
-        (fully_correlated_conditional_repeat, 5),
-        (fully_correlated_conditional_repeat, 1),
-        (fully_correlated_conditional, 1),
-    ],
+@pytest.fixture(
+    name="q_sqrt_factory", params=[lambda _, __: None, lambda LM, R: tf.eye(LM, batch_shape=(R,))]
 )
-def test_fully_correlated_conditional_repeat_shapes(func, R):
+def _q_sqrt_factory_fixture(request):
+    return request.param
+
+
+@pytest.fixture(name="full_cov", params=[True, False])
+def _full_cov_fixture(request):
+    return request.param
+
+
+@pytest.fixture(name="full_output_cov", params=[True, False])
+def _full_output_cov_fixture(request):
+    return request.param
```

Comment:
In another PR I saw this moved into a conftest.py - do you want to do that here already ?

Reply:
I can do that.

Reply:
Done